eng/Versions.props

```diff
@@ -7,8 +7,8 @@
   <PropertyGroup>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>5</MinorVersion>
-    <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>6</PreReleaseVersionLabel>
+    <PatchVersion>1</PatchVersion>
```

Comment:
GA is 4.5.0, right? Also not sure why we are changing the prerelease version here, it's only used as part of prerelease packages

Reply:
I don't know : ) basically I just follow this PR https://github.com/dotnet/roslyn/pull/57573